Packs/Core/Playbooks/playbook-Compromise_Accounts_-_User_rejected_numerous_SSO_MFA_attempts.yml

```diff
@@ -0,0 +1,1658 @@
+id: Compromise Accounts - User rejected numerous SSO MFA attempts
+version: -1
+name: Compromise Accounts - User rejected numerous SSO MFA attempts
+description: |-
+  This playbook addresses the following alerts:
+
+  - User rejected numerous SSO MFA attempts
+  - Multiple SSO MFA attempts were rejected by a user with suspicious characteristics
+
+  Playbook Stages:
+
+  Triage:
+  - The playbook checks the IP reputation associated with the MFA attempts and gathers related login events.
+
+  Early Containment:
+  - If the IP is identified as malicious, the playbook blocks the IP. The investigation continues in parallel to this phase.
```

Comment:
If the IP address is identified as malicious, the playbook blocks the IP. The investigation continues in parallel to this phase.
Comment